lib/middleware/websocket/interactors/payloads/race_payload.rb

```diff
@@ -1,23 +1,34 @@
-require './lib/typinggame_server/interactors/players_rooms/fetch_players_rooms'
+require './lib/typinggame_server/interactors/players_rooms/fetch_players_names_positions'
 
 module Websocket
   module Interactor
     class RacePayload
       def call(room_id:)
-        player_room_records =
-          Interactors::PlayersRooms::FetchPlayersRooms.new.call(
-            room_id: room_id
-          )
-            .player_room_records
+        players_names_positions =
+          fetch_players_names_positions(room_id: room_id)
 
         players = []
 
-        player_room_records.each do |player|
-          players << { 'id' => player.player_id, 'position' => player.position }
+        players_names_positions.each do |player|
```

Comment:
I believ we could use a `map` here instead of an `each` that pushes to an array.